pep-0639.rst

```diff
@@ -469,13 +425,23 @@ version.
 Add ``License-File`` field
 ''''''''''''''''''''''''''
 
-The ``License-File`` optional field is specified to contain the string
-representation of the path to a license-related file, relative to the
-root license directory. It is a multi-use field that may appear zero or
+Each instance of the ``License-File`` optional field is specified to contain
+the string representation of the path in the project source tree, relative to
+the project root directory, of a license-related file.
+It is a multi-use field that may appear zero or
 more times, each instance listing the path to one such file. Files specified
 under this field could include license text, author/attribution information,
 or other legal notices that need to be distributed with the package.
 
+As `specified by this PEP <License files in project formats_>`_, its value
+is also that file's path relative to the root license directory in both
+installed projects and the standardized distribution package types.
+In other legacy, non-standard or new distribution package formats and
+mechanisms of accessing and storing core metadata, the value MAY correspond
+to the license file path relative to a format-defined root license directory,
+or MAY be treated as a unique abstract key to access the license file contents
+by another means as specified by the format.
```

Comment:
Comment from a non expert -- this paragraph is a little hard to parse. Detailed specifications perhaps matter more, though!

Reply:
I rewrote it a bit to split into multiple sentences, add a comma and simplify it a touch.